src/main/java/com/indeed/jiraactions/JiraActionsIndexBuilder.java

```diff
@@ -40,7 +41,15 @@ public void run() throws Exception {
             final CustomFieldApiParser customFieldApiParser = new CustomFieldApiParser(userLookupService);
             final ActionFactory actionFactory = new ActionFactory(userLookupService, customFieldApiParser, config);
 
-            final IssuesAPICaller issuesAPICaller = new IssuesAPICaller(config, apiCaller);
+            boolean buildJiraIssuesApi = false;
+            final Stopwatch downloadStopwatch = Stopwatch.createStarted();
+            if(config.buildJiraIssues() ) {
+                final JiraIssuesIndexBuilder jiraIssuesIndexBuilder = new JiraIssuesIndexBuilder(config, new ArrayList<>(), new ArrayList<>());
+                buildJiraIssuesApi = jiraIssuesIndexBuilder.downloadTsv();
```

Comment:
We need a way to detect if the previous day's TSV is missing, so we download it first